components/brave_wallet_ui/common/hooks/assets-management.ts

```diff
@@ -50,14 +52,35 @@ export default function useAssetManagement () {
     }
   }
 
+  const addHiddenTokenToLocalStorage = (token: BraveWallet.BlockchainToken) => {
+    const assetId = getAssetIdKey(token)
+    const localStorageKey = token.isNft || token.isErc1155 || token.isErc721 
+      ? LOCAL_STORAGE_KEYS.USER_HIDDEN_NON_FUNGIBLE_TOKENS
+      : LOCAL_STORAGE_KEYS.USER_HIDDEN_FUNGIBLE_TOKENS
```

Comment:
Why not have a single key for both fungible and non-fungible tokens?

Reply:
Nit: use `USER_HIDDEN_TOKEN_IDS` as the key (and also change the actual value).

Reply:
@muliswilliam needs these separated for another feature! I have updated the `key` name though.